tests/wpt/metadata/webvtt/api/VTTCue/constructor.html.ini

```diff
@@ -0,0 +1,14 @@
+[constructor.html]
+  expected: CRASH
```

Comment:
What's the source of this CRASH right now?

Reply:
It's just a "not yet implemented crash" because `getCueAsHTML()` is called and there is a `todo!()` in its implementation.

Reply:
How is it being called right now if the interfaces are gated by the dom.webvtt.enabled preference, which is disabled by default?

Reply:
I passed `--pref dom.webvtt.enabled=true` to 'test-wpt` when testing the interfaces.